lib/chartmogul/subscription.rb

```diff
@@ -54,7 +54,7 @@ class Subscriptions < APIResource
     set_resource_root_key :subscriptions
 
     include Concerns::Entries
-    include Concerns::Pageable2
+    include Concerns::PageableWithCursor
```

Comment:
Why other resources have `next` method defined and this does not?

Reply:
The `next` method was there from previous versions and I didn't want to remove it to keep parity, should we then add it to the other resources?